Bugzilla/Bug.pm

```diff
@@ -279,6 +279,7 @@ use constant FIELD_MAP => {
   is_cc_accessible      => 'cclist_accessible',
   is_creator_accessible => 'reporter_accessible',
   last_change_time      => 'delta_ts',
+  last_change_time_non_bot => 'delta_ts_non_bot',
```

Comment:
Nit: it seems like the entire `FIELD_MAP` should have it's indentation updated here.